Source/WebKit/UIProcess/API/C/WKPage.cpp

```diff
@@ -3188,3 +3188,10 @@ void WKPageDispatchActivityStateUpdateForTesting(WKPageRef pageRef)
     CRASH_IF_SUSPENDED;
     toImpl(pageRef)->dispatchActivityStateUpdateForTesting();
 }
+
+#if ENABLE(NOTIFICATIONS)
```

Comment:
This isn't quite right, the `#if` check should be inside the function here since this is SPI and the function is not guarded in the header:
```
void WKPageClearNotificationPermissionState(WKPageRef pageRef)
 {
#if ENABLE(NOTIFICATIONS)
     toImpl(pageRef)->clearNotificationPermissionState();
#endif
 }
```

Reply:
That makes sense. Fixed!